procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb

```diff
@@ -5,10 +5,10 @@
    "id": "3dbf3c0c",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.        ingest_event_time = Time(ingest_event.private_sndStamp, format="unix", scale="tai")
</pre><p>you use these lines in at least 3 places in this event, if it's required, then make it a function, then call that function each time you need it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/92/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='65'/>

Reply:
Replaced.

Reply:
<p>See comment above in the functional definition.</p>

Reply:
<p>I guess I'm confused as to why the time is necessary....</p><p> </p>

Reply:
<p>In order to present some readable output to the person executing the notebook. </p>